core/storage/question/gae_models.py

```diff
@@ -248,24 +248,24 @@ def get_models_by_skill_id(cls, skill_id):
 
     @classmethod
     def get_models_by_question_id(cls, question_id):
-        """Returns a list of QuestionSkillLink domains of a particular
+        """Returns a list of QuestionSkillLinkModels of a particular
```

Comment:
> Does this correctly differentiate between "question ID doesn't exist" and "there are no question skill link models corresponding to this question ID"? Looking at the implementation I'm not sure it does, we might need to update the docstring to be clearer here.

Following on from my previous comment above, this needs backend tests:
- for the usual case
- for the case where the question ID is invalid
- for the case where the question ID is valid but there are no skill IDs that correspond to it

to verify that this behaves according to the specification in the docstring.

Reply:
Done, but I don't think there's an expected difference in output for the second and third point, as they should both return an empty list. Within the context of the QuestionSkillLinkModel the conditions for the second and third case are the same: no entity exists with that question ID.